new-client/src/plugins/Print/PrintModel.js

```diff
@@ -953,13 +970,21 @@ export default class PrintModel {
 
       // Add potential margin around the image
       if (this.margin > 0) {
-        // The lineWidth increases the line width equally to "both sides",
-        // therefore, we must have a line width two times the margin we want.
-        pdf.setLineWidth(this.margin * 2);
         // We always want a white margin
         pdf.setDrawColor("white");
-        // Draw the border (margin) around the entire image
-        pdf.rect(0, 0, dim[0], dim[1], "S");
+        // The lineWidth increases the line width equally to "both sides",
```

Comment:
Move this comment so that it makes sense and try to describe whats going on with A5 case.

Reply:
Sir, yes sir!